packages/s/sdl1-net/package.yml

```diff
@@ -1,23 +1,26 @@
 name       : sdl1-net
 version    : 1.2.8
-release    : 3
+release    : 4
 source     :
     - https://www.libsdl.org/projects/SDL_net/release/SDL_net-1.2.8.tar.gz : 5f4a7a8bb884f793c278ac3f3713be41980c5eedccecff0260411347714facb4
+homepage   : https://www.libsdl.org/
 license    : Zlib
 component  : multimedia.library
 summary    : SDL1 Networking
 description: |
     SDL1 Networking
 builddeps  :
-    - pkgconfig(sdl)
+    - pkgconfig32(liblzma)
+    - pkgconfig32(libzstd)
     - pkgconfig32(sdl)
+    - pkgconfig(sdl)
```

Comment:
`pkgconfig(sdl)` is not necessary because it is a dependency of `pkgconfig32(sdl)`